src/pokenav_unk_8.c

```diff
@@ -172,18 +195,516 @@ u32 sub_81CF030(struct PokenavSub7 *structPtr)
         return 0;
 }
 
-u32 sub_81CF0B0(struct PokenavSub7 *structPtr)
+static u32 sub_81CF0B0(struct PokenavSub7 *structPtr)
 {
     return 0x186A3;
 }
 
-u32 sub_81CF0B8(struct PokenavSub7 *structPtr)
+static u32 sub_81CF0B8(struct PokenavSub7 *structPtr)
 {
     return 0x186A9;
 }
 
-u32 sub_81CF0C0(void)
+static u32 sub_81CF0C0(void)
 {
     struct PokenavSub7 *structPtr = GetSubstructPtr(7);
     return structPtr->unk18;
 }
+
+static struct PokenavMonList * sub_81CF0D0(void)
+{
+    struct PokenavSub7 * ptr = (struct PokenavSub7 *)GetSubstructPtr(7);
```

Comment:
The casts aren't really necessary there since the return type is void*

Reply:
Fixed